tests/cross-language/java/scripts/build.sh

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env bash
+mvn -U compile
```

Comment:
we shouldn't do `-U`. The cross language tests should use the `appencryption` jar from the restored cache.
`-U` forces an update on all the snapshots

Reply:
we've used `mvn -U` in all compile scripts and some of them may possibly need to be changed as well. Will club this change along with those